Core/scripts/dirac-install.py

```diff
@@ -843,11 +843,17 @@ def downloadAndExtractTarball( tarsURL, pkgName, pkgVer, checkHash = True, cache
   tarName = "%s-%s.tar.gz" % ( pkgName, pkgVer )
   tarPath = os.path.join( cliParams.targetPath, tarName )
   tarFileURL = "%s/%s" % ( tarsURL, tarName )
+  tarFileCVMFS = "/cvmfs/dirac.egi.eu/installSource/%s" % tarName
```

Comment:
I agree with Federico that if you want to manage your own repository also for DIRAC codes you have to create your own command to do it your way. You can follow or not the LHCb pattern. Or you can just follow the DIRAC pattern with another, configurable, CVMFS repo.